lstchain/scripts/lstchain_check_dl1.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+"""
+Script to check the contents of LST DL1 files and associated muon ring files
+To run it, type e.g.:
+python lstchain_check_dl1.py
+--input_file dl1_LST-1.1.Run01881.0000.h5
+"""
+from warnings import simplefilter
+import argparse
+import glob
+# I had enough of those annoying future warnings, hence:
+simplefilter(action='ignore', category=FutureWarning)
+from lstchain.datachecks import check_dl1, plot_datacheck
+
+parser = argparse.ArgumentParser(formatter_class=argparse.
+                                 ArgumentDefaultsHelpFormatter)
+
+# Required arguments
+# input file(s). Wildcards can be used, but inside quotes e.g. "dl1*.h5"
+parser.add_argument('--input_file', type=str, required=True,
+                    help='Path to DL1 data file(s) (containing pixel-wise '
+                         'charge information and image parameters)'
+                    )
+
+# Optional arguments
+# path for output files
+parser.add_argument('--output_path', default='.', type=str,
+                    help='Path to the output files'
+                    )
+# maximum number of processes to be run in parallel
+# This refers to the processes explicitly spawned by check_dl1, not to what
+# e.g. numpy may do on its own!
+parser.add_argument('--max_cores', default=4, type=int,
+                    help='Maximum number of processes spawned'
+                    )
+
+
+args = parser.parse_args()
+
+
+def main():
+
+    print('input files: {}'.format(args.input_file))
+    print('output path: {}'.format(args.output_path))
+
+    filenames = glob.glob(args.input_file)
+    if len(filenames) == 0:
+        print('Input files not found!')
```

Comment:
Use raise error or logging error instead of print?

Reply:
> Use raise error or logging error instead of print?

Ok, using logging now (but prefer to switch off the extra logging messages for now, module, type of message... - eventually we can switch on via command line but I do not know if we have any standard for doing so)